### PR TITLE
App FQDN depends on the region

### LINKED ISSUE #696: Domain name should point to the region subdomain
Today our doc is now wrong: CNAME/A records should not point to subdomain.scalingo.io, they should point to subdomain.region.scalingo.io

Example: https://doc.scalingo.com/platform/app/domain#configure-your-domain-name).

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #696

### CODE REVIEW

File: _posts/platform/app/2000-01-01-domain.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ tags: app domain-name dns cname http
 index: 20
 ---
 
-Scalingo provides a subdomain of `scalingo.io` for all applications to access your application.
+Scalingo provides a subdomain of `osc-fr1.scalingo.io` for all applications to access your application.
```
Comment (yannski) [inline _posts/platform/app/2000-01-01-domain.md]:
```suggestion
Scalingo provides a subdomain looking like `region-name.scalingo.io` for all applications to access your application.
```

File: _posts/platform/app/2000-01-01-domain.md [hunk 2]
```diff
@@ -54,16 +54,20 @@ by clicking on 'Link domain name to the app'.
 After having bought a domain name you need to configure it. Most of the
 registrars provide a web dashboard to do so. To target your application
 hosted on Scalingo, you need to create a `CNAME` field targeting the
-fully qualified domain name (FQDN) `my-app.scalingo.io.`
+fully qualified domain name (FQDN). This FQDN depends on the region your
+application is deployed on:
+
+- `agora-fr1`: `my-app.scalingo.io.`
```
Comment (yannski) [inline _posts/platform/app/2000-01-01-domain.md]:
```suggestion
- `agora-fr1`: `my-app.agora-fr1.scalingo.io.`
```

File: _posts/platform/app/2000-01-01-ssl.md [hunk 3]
```diff
@@ -9,7 +9,8 @@ index: 21
 ## \*.scalingo.io Domain
 
 By default you can access your application using the following domain name:
-my-app.scalingo.io.
+`my-app.<region>.scalingo.io`. Replace `<region>` with the region where is your
```
Comment (yannski) [inline _posts/platform/app/2000-01-01-ssl.md]:
```suggestion
`my-app.<region>.scalingo.io`. Replace `<region>` with the region where you have created your
```

File: _posts/platform/app/2000-01-01-domain.md (hunk 2 above)
Comment (EtienneM) [reply _posts/platform/app/2000-01-01-domain.md]: This one does not work

**Comment by yannski** [comment]
LGTM

**Pull request merged**